vignettes/InstantaneousTimeStep.Rnw

```diff
@@ -23,10 +30,10 @@ This example uses data from the Bad River near Odanah, Wisc., USGS gaging statio
 library(rloadest)
 library(dataRetrieval)
 # What unit values are available?
-subset(whatNWISdata("04027000", "uv"),
-  select=c("parm_cd", "srsname", "begin_date", "end_date"))
+# subset(whatNWISdata("04027000", "uv"),
+#   select=c("parm_cd", "srsname", "begin_date", "end_date"))
```

Comment:
not sure why the old version didn't work, but rather than commenting this out, how about
```r
subset(whatNWISdata("04027000"), data_type_cd=="uv",
   select=c("parm_cd", "srsname", "begin_date", "end_date"))
```
instead?